runs/2017/mipyt-kam/link.yml

```diff
@@ -1,2 +1,2 @@
-repo: https://github.com/cvut/naucse.python.cz.git
-branch: b171
+repo: https://github.com/pyvec/naucse.python.cz/
```

Comment:
why the change in URL?

Reply:
Hmm, I didn't think about this, to be honest – I just added the pyvec URL everywhere, and it turns out I changed the existing repo names. Is it a problem?

Generally I like `/` rather than `.git` a bit better since you can just add e.g. `pulls/` to the end and get a valid URL.

Reply:
I don't care either way, I was just curious.